pages/lobby.js

```diff
@@ -0,0 +1,202 @@
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    await axios.post("/api/pusher", {
+      message: messageToSend,
+      username,
+    });
+    setMessageToSend("");
+  };
+
+  const handleJoinPlayer = (event) => {
+    event.preventDefault();
+    const hostName = event.target.elements[0].value;
+
+    router.push(`/multiplayer/${hostName}`);
+  };
+
+  return (
+    <>
+      <h2>
+        TOTALLY <i>ZOMBIFIED</i> CHAT
+      </h2>
+      <div>
+        <strong>{username}</strong>
+        <StyledButton onClick={handleSignOut}>Sign out</StyledButton>
+        <StyledChat>
+          <h2>GAMELOBBY</h2>
+          <div>
+            <h4> {onlineUsersCount} user(s) online now</h4>
+          </div>
+          <div>
+            {/* show online users */}
+            {onlineUsers.map((user, id) => (
+              <div key={id}>
+                <small>
+                  {" "}
+                  <span>{user.username}</span> joined the chat!
+                </small>
+              </div>
+            ))}
+            {/* show users leaving the chat */}
+            {usersRemoved.map((user, id) => (
+              <div key={id}>
+                <small>
+                  {" "}
+                  <span>{user}</span> left the chat.
+                </small>
+              </div>
+            ))}
+          </div>
+
+          <div>
+            {chats.map((chat, id) => (
+              <StyledMessage key={id}>
+                <small>{chat.username}:</small> {chat.message}
+              </StyledMessage>
+            ))}
+          </div>
+
+          <div>
+            <form onSubmit={handleSubmit}>
+              <StyledInput
+                type="text"
+                value={messageToSend}
+                onChange={(event) => setMessageToSend(event.target.value)}
+                placeholder="start typing...."
+              />
+              <StyledButton type="submit">Send</StyledButton>
+            </form>
+          </div>
+        </StyledChat>
+      </div>
+      <div>
+        PLAYER ONE: <br />
+        <Link href={`/multiplayer/${username}`}>
+          <StyledButton type="text">Create Game</StyledButton>
+        </Link>
+        <br />
+        PLAYER TWO: <br />
+        <form onSubmit={handleJoinPlayer}>
+          <StyledInput type="text" placeholder="Player you want to join..." />
+          <StyledButton type="submit">Join Player</StyledButton>
+        </form>
+      </div>
+    </>
+  );
+}
+
+const StyledButton = styled.button`
+  text-align: center;
+  font-weight: bold;
+  color: black;
+  background-color: beige;
+  border: solid black 0.2rem;
+  border-radius: 5px;
+  margin: 0.2rem;
+  padding: 0.5rem 1rem;
+`;
+
+const StyledChat = styled.section`
+  text-align: center;
+  background-color: black;
+  color: white;
+  border-radius: 5px;
+  padding: 10px 1rem;
+  margin: 5px 10px;
+  h2 {
+    margin: 0;
+    font-weight: 900;
+    font-size: x-large;
+    letter-spacing: 5px;
+  }
+  h4 {
```

Comment:
since you'll change it above as well ;) 

```suggestion
  h3 {
```